src/handlers/int-test-support/helpers/athenaHelper.ts

```diff
@@ -57,6 +57,8 @@ const getQueryExecutionStatus = async (
   const response = await athenaClient.send(
     new GetQueryExecutionCommand(params)
   );
+
+  console.log(response.QueryExecution?.Status);
```

Comment:
I would like to keep this console.log for few days to see the error its logs as I have seen  query failure intermittently when running tests in github actions but not able to replicate locally